roles/users/README.md

```diff
@@ -0,0 +1,149 @@
+# users
+
+## This role can be used to create users, user groups, LDAP-Auth and to link external and internal user groups
+
+### You can disable certain tasks
+
+#### Tasks to switch on/off
+
++ create_users
++ create_usergroups
++ enable_ldap_auth
++ map_external_usergroups
+
+### Role Variables
+
+#### Example values can be found in defaults/main.yaml
+
+Default:
+
+``` yaml
+server_url: string(url) **required**
+username: string **required**
+password: string
+password: string
+validate_certs: bool defualt (true)
+```
+
+To switch on/off tasks use:
+
+``` yaml
+create_users: bool defualt (true)
+create_usergroups: bool defualt (true)
+enable_ldap_auth: bool defualt (true)
+map_external_usergroups: bool defualt (true)
```

Comment:
```suggestion
create_users: bool default (true)
create_usergroups: bool default (true)
enable_ldap_auth: bool default (true)
map_external_usergroups: bool default (true)
```

Reply:
What is the usecase here? Would it make sense to just perform the actions if the corresponding data structures are not empty?

Reply:
I have a full playbook (vars) what describes my instance or in this case my user space, with the above vars in a playbook i can choose what to run without del/paste data.

Reply:
I'm not sold on this. Do we have a similar thing in other roles?

Reply:
To be a bit more verbose: I thought, the deal of this role is to define your instance and the role will do every thing needed. I don't understand, why i should specify everything to then not use part of it. I could still be convinced though.

Maybe as a guideline: Those roles fall into the 80/20 rule. If you need something really special, you can write your own. We want to serve the common use case.

Reply:
You dont have to specify anything, only the data for enabled tasks.

I think this happens very often as an admin. Example: 
If an admin already has the playbook written and wants to add an internal user without going to the whole process with ldap etc. he can only activate the create user part and run it. 
On the other side you can only create a auth source and link groups without creating plenty user(s).